spec/factories/article_factory.rb

```diff
@@ -0,0 +1,8 @@
+FactoryGirl.define do
+  factory :article do
+    sequence(:title) { |n| "article#{n}"}
+    body 'This is the body'
+    author_name 'Connor'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.